src/vireshJJun20/Assignment17/IndexCheck.java

```diff
@@ -0,0 +1,37 @@
+package vireshJJun20.Assignment17;
+
+public class IndexCheck {
+
+	void checkIndex(String str) { // kindly ignore this method, basic method used as a reference for better understanding
+		for (int i = 0; i < str.length(); i++) {
+
+			char ch1 = str.charAt(i);
+			int temp = str.indexOf(ch1);
+			System.out.println(ch1 + " -> First index = " + temp + " Last index = " + str.lastIndexOf(ch1));
+
+		}
+	}
+
+	void checkIndex1(String str) { // This method gives the expected output, but not sure about the logic if its good.
+		
+		for(int i=0; i<str.length();i++) {
+			char ch1= str.charAt(i);
+			int ch1Cnt=str.indexOf(ch1);
+			//int cnt=0;
+			if (i== str.indexOf(ch1)) {
```

Comment:
Here one more and condition required with if block that is str.indexOf(ch) != str.lastIndexOf(ch)
Try this and commit code.

Reply:
Yes.. that was missing, I had added it and it worked perfectly. Thanks Krishna.

![image](https://user-images.githubusercontent.com/66830247/85831873-9f1da700-b7ac-11ea-9a84-60e3a10424fb.png)